### ISSUE TITLE
Cannot add multiple accounts with identical username

### ISSUE THREAD

**Issue by thefranke** [brentsimmons/NetNewsWire#4092]
Usually when setting up accounts, one tends to choose the same username for different services.

In my case, I have two FreshRSS instances set up with identical usernames. NetNewsWire though complains that it cannot add the second account because the credentials are identical. Is there a reason for this?

**Comment by brentsimmons** [comment]
When we first started adding sync accounts, I was thinking of services such as Feedbin and Feedly where each username is, and has to be, unique.

The code that checks the credentials is there to prevent a person from adding two of the exact same accounts (same service and same username), since that’s not something a user would want to happen.

The idea that a person might subscribe to multiple instances of FreshRSS with the same username is just not a thing I thought of.

**Comment by thefranke** [comment]
Makes sense. For now I stumble into this issue every now and then when registering for an account and accidentally using the same name. Not a big issue since one can re-register, only an annoyance. Would be nice to have this restriction loosened in the future.

**Comment by brentsimmons** [comment]
Done in 7.1.3.
